web/thesauruses/clojure/1/operators.json

```diff
@@ -0,0 +1,223 @@
+{
+  "meta": {
+    "language": "clojure",
+    "language_version": "1.10.3",
+    "language_name": "Clojure",
+    "structure": "operators"
+  },
+  "concepts": {
+    "addition": {
+      "code": "(+ 1 2 3)",
+      "name": "Addition operator"
+    },
+    "addition_assignment": {
+      "comment": "Clojure doesn't have the traditional notion of assignment as in procedural languages, but it can apply in some situations (ex: atoms and volatile refs).",
+      "code": "(let [x (volatile! 0)]\n  (doseq [addend (range 10)]\n    (vswap! x + addend))\n  @x)",
```

Comment:
This is basically inventing the solution, and Code Thesaurus is a reference tool for what exists. So therefore it should probably be "not-implemented". (Same for similar assignments below)